apps/condo/domains/resident/schema/RegisterServiceConsumerService.js

```diff
@@ -50,13 +50,15 @@ const ERRORS = {
         code: BAD_USER_INPUT,
         type: NOT_FOUND,
         message: 'Can\'t find billingAccount and any meters with this accountNumber, unitName and address',
+        messageForUser: 'api.billing.RegisterServiceConsumerService.BILLING_ACCOUNT_OR_METER_NOT_FOUND',
```

Comment:
This is the domain `resident`
```suggestion
        messageForUser: 'api.resident.RegisterServiceConsumerService.BILLING_ACCOUNT_OR_METER_NOT_FOUND',
```